doc/sections/options.md

```diff
@@ -1,134 +1,134 @@
+
+ - `--timeout NUM` gives up type-checking after the given number of seconds.
    Note that the timeout is reset between each file, and that the parameter of
    the command is expected to be a natural number.
- - `--too-long <float>` gives a warning for each command (i.e., file item) taking
-   more than the given number of seconds to be checked. The given parameter is
-   expected to be a floating point number.
 
-#### Rewriting engine
-
-The following options can be used to modify the behaviour of the
-reduction engine,
- - `--write-trees` writes the decision trees used for rule filtering
-   to dot files; for each symbol `s`, a file `mod_path/mod_name.s.gv` is
+ - `--write-trees` writes the decision trees used for rewriting rule filtering
+   to dot files. For each symbol `s`, a file `mod_path/mod_name.s.gv` is
    generated. For more information on trees (and how to read them), see
    `src/tree_graphviz.ml`.
- - `--keep-rule-order` forces the rewriting engine to use in priority
-   the topmost rules.
+
+##### Confluence checking
+
+Lambdapi provides an option `--confluence CMD` to check the confluence of the
+rewriting system by calling an external prover with the command `CMD`. The
+given command receives [TRS](http://project-coco.uibk.ac.at/problems/trs.php)
+formatted text on its standard input, and it is expected to output on the
+first line of its standard output either `YES`, `NO` or `MAYBE`.
+
+As an example, `echo MAYBE` is the simplest possible (valid) confluence-check
+that one may use.
+
+For now, only the [`CSI^ho`](http://cl-informatik.uibk.ac.at/software/csi/ho/)
+confluence checker has been tested with Lambdapi. It can be called using the
+flag `--confluence "path/to/csiho.sh --ext trs --stdin"`.
+
+To inspect the `.trs` file generated by Lambdapi, one may use the following
+dummy command: `--confluence "cat > output.trs; echo MAYBE"`.
+
+##### Termination checking
+
+Lambdapi provides an option `--termination CMD` to check the termination of
+the rewriting system by calling an external prover with the command `CMD`. The
+given command receives [XTC](http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB/raw-file/tip/xml/xtc.xsd)
+formatted text on its standard input, and it is expected to output on the
+first line of its standard output either `YES`, `NO` or `MAYBE`.
+
+As for confluence, `echo MAYBE` is the simplest possible (valid) command for
+checking termination.
+
+To the best of our knowledge, the only termination checker that is compatible
+with all the features of Lambdapi is [`SizeChangeTool`](https://github.com/Deducteam/SizeChangeTool).
+It can be called using the flag `--termination "path/to/sct.native --no-color --stdin=xml"`
+
+If no type-level rewriting is used [`Wanda`](http://wandahot.sourceforge.net/)
+can also be used. However, it does not directly accept input on its standard
+input, so it is tricky to have Lambdapi call it directly. Alternatively, one
+can first generate a `.xml` file as described below.
```

Comment:
below where?

Reply:
Literally the next line.

Reply:
Then, I propose:

If there is no type-level rewriting rules, then [`Wanda`](http://wandahot.sourceforge.net/) can also be used.

For provers like Wanda that do not read the standard input, one needs to generate a XTC file first as follows: `--termination "cat > output.trs; echo MAYBE"`.

Reply:
Doing that makes this paragraph less consistent with what I wrote for confluence.

Don't you think we could maybe improve on the doc later? I've clearly made a lot of improvements on the doc, actually a lot more than I should have since they were not directly related to what I did. Maybe you can see with @GuillaumeGen  later if he can clarify the doc. In particular, I have been meaning to try something like the following command for Wanda to see if it works: `--termination "cat > /tmp/output.xml; wanda output.xml"`. If it does, the doc can be made clearer by giving it directly.